packages/remix-dev/vite/plugin.ts

```diff
@@ -120,6 +122,13 @@ export type RemixVitePluginOptions = RemixConfigJsdocOverrides &
      * as a SPA without server-rendering. Default's to `true`.
      */
     unstable_ssr?: boolean;
+    /**
+     * Specify a JS filename to enable "library mode" which will write out a
+     * JS file instead of an `index.html` file so you can embed your Remix SPA
+     * Mode app as a library in an HTML page generated elsewhere.  Requires
+     * `unstable_ssr: false`.
+     */
+    unstable_library?: string;
```

Comment:
When in spa mode, users can specify an `unstable_library 'app.js'` config to specify they want the app written out as a JS file they can embed into HTML generated by some other system.